Dockerfile

```diff
@@ -0,0 +1,53 @@
+### Note: This DockerFile is adapted from https://github.com/ucb-bar/chipyard/blob/master/.circleci/images/Dockerfile which was adapted from: https://github.com/CircleCI-Public/example-images/openjdk
+# This is a full chipyard setup, which will be built manually on-demand in the Tendermint Hardware Project at https://gitlab.com/virgohardware/core/
+
+FROM ubuntu:18.04
+
+MAINTAINER jacobgadikian@gmail.com
+
+# man directory is missing in some base images
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+               curl \
+               git \
+               sudo
+
+# Stopping docker keyboard-config from disrupting ubuntu-req.sh
+RUN sudo DEBIAN_FRONTEND=noninteractive apt-get install -y keyboard-configuration && \
+   sudo DEBIAN_FRONTEND=noninteractive apt-get install -y console-setup
+
+RUN groupadd --gid 3434 riscvuser \
```

Comment:
Can you add a small comment here explaining that this adds a new user called `riscvuser`